dMasterServer/MasterServer.cpp

```diff
@@ -143,26 +143,38 @@ int main(int argc, char** argv) {
 
 	MigrationRunner::RunMigrations();
 
-	// Check CDClient exists
-	if (!std::filesystem::exists(Game::assetManager->GetResPath() / "CDServer.sqlite")) {
-		Game::logger->Log("WorldServer", "CDServer.sqlite could not be opened. Looking for cdclient.fdb to convert to sqlite.");
-
-		if (!std::filesystem::exists(Game::assetManager->GetResPath() / "cdclient.fdb")) {
-			Game::logger->Log("WorldServer", "cdclient.fdb could not be opened. Please move a cdclient.fdb or an already converted database to build/res.");
-			return EXIT_FAILURE;
-		}
-
-		Game::logger->Log("WorldServer", "Found cdclient.fdb.  Converting to SQLite");
-
-		if (FdbToSqlite::Convert(Game::assetManager->GetResPath().string()).ConvertDatabase() == false) {
-			Game::logger->Log("MasterServer", "Failed to convert fdb to sqlite");
-			return EXIT_FAILURE;
+	const bool cdServerExists = std::filesystem::exists(BinaryPathFinder::GetBinaryDir() / "resServer" / "CDServer.sqlite");
+	const bool oldCDServerExists = std::filesystem::exists(Game::assetManager->GetResPath() / "CDServer.sqlite");
+	const bool fdbExists = std::filesystem::exists(Game::assetManager->GetResPath() / "cdclient.fdb");
+
+	if (!cdServerExists) {
+		if (!oldCDServerExists) {
```

Comment:
Can you swap the branches and remove the `!`?

Reply:
yes - will do so in the morning